docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+
+  indexer:
+      build:
+        context: .
+        dockerfile: Dockerfile
+      ports:
+        - "8080:42069"
```

Comment:
can we rely on the ponder defaults and instead of specifying a port to that container we just map `42069:42069`?